odbc/tests/common/mod.rs

```diff
@@ -1,51 +1 @@
-use lazy_static::lazy_static;
```

Comment:
The `odbc` crate doesn't allow for returning a connection  since it is tied to the lifetime of the `env`.